experiments/ClimaEarth/leaderboard/leaderboard.jl

```diff
@@ -136,6 +142,7 @@ function compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
     end
 
     # Add RMSE for the CliMA model and for each season
+    rmse_var_dict = filter(((k, _),) -> k in keys(sim_var_dict), rmse_var_dict)
```

Comment:
This takes a minute to process. Could you maybe move the anonymous function outside of the filter and give it a name so that the intent is a little bit more clear?